kuma/wiki/search.py

```diff
@@ -31,25 +30,25 @@ class WikiDocumentType(document.DocType):
                      'Project_talk:', EXPERIMENT_TITLE_PREFIX]
 
     boost = field.Float(null_value=1.0)
-    content = field.String(analyzer='kuma_content',
-                           term_vector='with_positions_offsets')
-    css_classnames = field.String(analyzer='case_insensitive_keyword')
-    html_attributes = field.String(analyzer='case_insensitive_keyword')
+    content = field.Text(analyzer='kuma_content',
+                         term_vector='with_positions_offsets')
+    css_classnames = field.Keyword()
+    html_attributes = field.Keyword()
     id = field.Long()
-    kumascript_macros = field.String(analyzer='case_insensitive_keyword')
-    locale = field.String(index='not_analyzed')
+    kumascript_macros = field.Keyword()
+    locale = field.Keyword()
     modified = field.Date()
     parent = field.Nested(properties={
```

Comment:
I strongly believe this should be `Object` datatype instead of `Nested` datatype. `Nested` datatype is preferrable for arrays of objects.

Reply:
OK will try it